Replace Launchpad with GitHub (#2345)
tarantool/doc: doc/dev_guide/developer_guidelines.rst

===========================================================

> Any defect, even minor, if it changes the user-visible server behavior, needs
> a bug report. Report a bug at

Reviewer: ```suggestion
 If a defect changes user-visible server behavior, it needs a bug report, even if it is a small defect. Report the bug at
```
